addons/reloadlaunchers/ACE_Arsenal_Stats.hpp

```diff
@@ -0,0 +1,13 @@
+class EGVAR(arsenal,stats) {
+    class statBase;
+    class ADDON: statBase {
+        scope = 2;
+        priority = -1;
+        stats[] = {QGVAR(enabled)};
+        displayName = CSTRING(LoadLauncher);
```

Comment:
This could easily be misinterpreted as an indication of the launcher being disposable. Alternate stringtable maybe? (`Buddy loading`?)